no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
```

Comment:
In this case, I'd suggest `packages` default to `[]`, and then there's no need for the check (and if it's not a list, an appropriate exception will be raised.

But generally, I'd recommend strongly against `if foo` in place of `if foo is not None`.